ansible/group_vars/all.yml.sample

```diff
@@ -18,3 +18,12 @@ dummy:
 #grafana:
 #  admin_user: admin
 #  admin_password: admin
+# You may want to change the default theme to dark
+#  default_theme: light
+# If you use email alerting, configuer your SMTP Connection. If your SMTP Server dos not require authentuicatin, skip smtp_user and smtp_password
```

Comment:
Fix the typos, please: s/configuer/configure/, s/dos/does/, s/authentuicatin/authentication/.

Reply:
Thanks for review. I have fixed the typos.